programmers-coding-test/minjae/Implement/구현-왕실의-게임.js

```diff
@@ -0,0 +1,25 @@
+function solution(n, m) {
+    const chessLength = 8;
```

Comment:
이런 스페셜(?) 상수는 전부 대문자로 해주는게 좋을 듯 합니다.
```suggestion
    const CHESS_LENGTH = 8;
```